dependencies.gradle

```diff
@@ -29,7 +29,7 @@ depVersions = [
     commonsCli: "1.2",
     commonsCodec: "1.6",
     commonsCollections4: "4.1",
-    commonsCompress: "1.21",
+    commonsCompress: "1.19",
```

Comment:
why this one? 
1.19 is vulnerable
we have 1.21 in Maven pom

Reply:
good catch. I picked up a old version of the file from master branch